exotic/exotic.py

```diff
@@ -3147,6 +3161,8 @@ def main():
             f"               Airmass coefficient 2: {round_to_2(myfit.parameters['a2'], myfit.errors['a2'])} +/- {round_to_2(myfit.errors['a2'])}")
         log.info(
             f"                    Residual scatter: {round_to_2(100. * np.std(myfit.residuals / np.median(myfit.data)))} %")
+        log.info(
+            f"              Transit Duration [day]: {np.mean(durs):.5f} +/- {np.std(durs):.5f}")
```

Comment:
please change to:
`"Transit Duration (day)":f"{round_to_2(np.mean(durs),np.std(durs))} +/- {round_to_2(np.std(durs))}"`

Reply:
Still gotta do this one, please!